core/domain/subscription_services.py

```diff
@@ -167,13 +171,16 @@ def get_all_threads_subscribed_to(user_id):
     """
     subscriptions_model = user_models.UserSubscriptionsModel.get(
         user_id, strict=False)
-
-    return (
-        subscriptions_model.general_feedback_thread_ids
-        if subscriptions_model else [])
+    if subscriptions_model:
+        feedback_thread_ids: (
```

Comment:
Ditto, as above. Try keeping the type in the same line

Reply:
Done!